src/playground/dist/views/main.handlebars

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
```

Comment:
Why did you pick Handlebars here instead of React or [Hybrids](https://github.com/hybridsjs/hybrids)?

Reply:
Because it's minimal. In fact, we could live even without any templates at all in this project, so I chose the most minimalistic solution.